dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/api/dto/PagerDTO.java

```diff
@@ -0,0 +1,262 @@
+    }
+
+    public void setPreviousPageNumber(Integer previousPageNumber) {
+        this.previousPageNumber = previousPageNumber;
+    }
+
+    public void setHasNextPageNumber(Boolean hasNextPageNumber) {
+        this.hasNextPageNumber = hasNextPageNumber;
+    }
+
+    public void setNextPageNumber(Integer nextPageNumber) {
+        this.nextPageNumber = nextPageNumber;
+    }
+
+    public void setStartCardNumber(Integer startCardNumber) {
+        this.startCardNumber = startCardNumber;
+    }
+
+    public void setEndCardNumber(Integer endCardNumber) {
+        this.endCardNumber = endCardNumber;
+    }
+
+    public void setStartCardNumberString(String startCardNumberString) {
+        this.startCardNumberString = startCardNumberString;
+    }
+
+    public void setEndCardNumberString(String endCardNumberString) {
+        this.endCardNumberString = endCardNumberString;
+    }
+
+    public void setRemainingCards(Integer remainingCards) {
+        this.remainingCards = remainingCards;
+    }
+
+    public void setStartResultNumber(Integer startResultNumber) {
+        this.startResultNumber = startResultNumber;
+    }
+
+    public void setEndResultNumber(Integer endResultNumber) {
+        this.endResultNumber = endResultNumber;
+    }
+
+    public void setStartResultNumberString(String startResultNumberString) {
+        this.startResultNumberString = startResultNumberString;
+    }
+
+    public void setEndResultNumberString(String endResultNumberString) {
+        this.endResultNumberString = endResultNumberString;
+    }
+
+    public void setRemainingResults(Integer remainingResults) {
+        this.remainingResults = remainingResults;
+    }
+
+    public void setNumberNextResults(Integer numberNextResults) {
+        this.numberNextResults = numberNextResults;
+    }
+
+    public void setPageNumberList(List<Integer> pageNumberList) {
+        this.pageNumberList = pageNumberList;
+    }
+
+    // -------------------- INNER CLASSES --------------------
+
+    public static class Converter {
+        public enum Mode { CARD, RESULT }
+
+        public PagerDTO convert(Pager pager, Mode mode) {
+            PagerDTO converted = new PagerDTO();
+            converted.setNecessary(pager.isPagerNecessary());
+            converted.setNumResults(pager.getNumResults());
+            converted.setNumResultsString(pager.addCommasToNumber(pager.getNumResults()));
+            converted.setDocsPerPage(pager.getDocsPerPage());
+            converted.setSelectedPageNumber(pager.getSelectedPageNumber());
+            converted.setPageCount(pager.getPageCount());
+            converted.setHasPreviousPageNumber(pager.hasPreviousPageNumber());
+            converted.setPreviousPageNumber(pager.getPreviousPageNumber());
+            converted.setHasNextPageNumber(pager.hasNextPageNumber());
+            converted.setNextPageNumber(pager.getNextPageNumber());
+
+            if (Mode.CARD.equals(mode)) {
```

Comment:
IMO distinction between modes adds only noise. As far as I can see `Pager` is used only in mydata and listing users (superuser only operation). Maybe we can take advantage that mydata endpoint was never documented and go with version that was used in listing users?
BTW: this whole `Pager` object is so pointless in exposing it via api - but this is a discussion for another time.